packages/ethereum-contracts/contracts/superfluid/SuperfluidToken.sol

```diff
@@ -407,6 +416,76 @@ abstract contract SuperfluidToken is ISuperfluidToken
             );
         }
     }
+    
+    /// @dev ISuperfluidToken.makeLiquidationPayoutsV2 implementation
+    function makeLiquidationPayoutsV2(
+        bytes32 id,
+        bytes memory liquidationTypeData,
+        address liquidatorAccount,
```

Comment:
add comments for explaining the different accounts + rewardRecipientAccount => rewardAccount